packages/auth/src/grant/service.test.ts

```diff
@@ -87,31 +87,13 @@ describe('Grant Service', (): void => {
       assetScale: 9
     },
     expiresAt: new Date().toISOString(),
-    description: 'this is a test',
+    description: 'this is a test you fuck',
```

Comment:
:eyes:

Reply:
Forgive my exasperation with getting the tests working 😓